ci/bosh-lite/check-broken-pool-size.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -eu
+
+# ENV
+: "${POOL_NAME:="broken-bosh-lites"}"
+: "${GIT_USERNAME:="CAPI CI"}"
```

Comment:
I know this doesn't matter much, but should this be referring to the new ARI Github bot?

https://github.com/cloudfoundry/capi-ci/blob/1429aa6b024e72e348bf78eb428b7c26e33cce82/ci/pipeline.yml#L133-L134